### PR TITLE
VIH-7367 judge and joh hearing list

### PR DESCRIPTION
**Pull request by valentinabennett**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/VIH-7367
### Change description ###
Hearing list for Judge and JOH accounts

**Does this PR introduce a breaking change?** (check one with "x")

```
[ X] Yes
[ ] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/Controllers/ConferencesController.cs [hunk 1]
```diff
@@ -53,10 +53,14 @@ public class ConferencesController : Controller
         [ProducesResponseType(typeof(List<ConferenceForJudgeResponse>), (int)HttpStatusCode.OK)]
         [ProducesResponseType((int)HttpStatusCode.BadRequest)]
         [SwaggerOperation(OperationId = "GetConferencesForJudge")]
-        [Authorize(AppRoles.JudgeRole)]
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/Controllers/ConferencesController.cs:56]: Please bring this back and use the judicial role instead or both

**Review by shaed-parkar — CHANGES_REQUESTED** [review]
Please keep the authorize attributes for the endpoints. It was a part of a ITHC piece a while ago. Add a policy for JOHs under `ConfigureAuthSchemeExtensions`.

File: VideoWeb/VideoWeb/Controllers/ConferencesController.cs (hunk 1 above)
Comment (valentinabennett) [reply VideoWeb/VideoWeb/Controllers/ConferencesController.cs:56]: Done, add new authorize policy

**Review by shaed-parkar — APPROVED** [review]
Thanks LGTM

**Pull request merged**